### PR TITLE
MAUI-3988 : Migration document for MAUI Maps.

### PR DESCRIPTION
**Pull request by Shalini-Ashokan**
Hi @vigneshrameshsync 
I have updated migration document for tile layer in MAUI Maps UG. Could you please review it and let me know if you have any concerns?

Regards,
Shalini Ashokan.

### CODE REVIEW

File: MAUI/Maps/getting-started.md [hunk 1]
```diff
@@ -267,6 +267,42 @@ public MainPage()
...
+    layer.Geometry = MapGeometryType.Points;
+    map.Layer = layer;
+    this.Content = map;
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![maps basic view](images/getting-started/maps_cartesian_view.png)
```
Comment (vigneshrameshsync) [inline MAUI/Maps/getting-started.md:304]: Currently we don't have multiple selection support. So, please change the output image without selection.

File: MAUI/Maps/migration.md [hunk 2]
```diff
@@ -349,6 +351,302 @@ The following code example explains how to intialize the layer and customize the
...
+   <td>
+      {{'[DistanceType]()'| markdownify }}
+   </td>
+</tr>
+<tr>
+   <td>
+      {{'[LatLngBounds](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfMaps.XForms.ImageryLayer.html#Syncfusion_SfMaps_XForms_ImageryLayer_LatLngBounds)'| markdownify }} 
+   </td>
+   <td>
+      {{'[MapLatLngBounds]()'| markdownify }}
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:430]: '*' missing

File: MAUI/Maps/migration.md [hunk 3]
```diff
@@ -349,6 +351,302 @@ The following code example explains how to intialize the layer and customize the
...
+   </td>
+   <td>
+      {{'[ZoomLevelChanging]()'| markdownify }} 
+   </td>
+</tr>
+</table>
+
+N> All the (*) marked APIs are renamed from Xamarin SfMaps to maintain the consistency of API naming in the .NET MAUI SfMaps.
+
+The following code example explains how to intialize the zooming and it's customization in Xamarin SfMaps and .NET MAUI SfMaps.
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:487]: The following code example explains how to initialize the tile layer and its customization in Xamarin SfMaps and .NET MAUI SfMaps.

File: MAUI/Maps/migration.md [hunk 4]
```diff
@@ -3040,17 +3386,365 @@ public MainPage()
 </td></tr>
 </table>
 
+## Vector layer
+
+### Polygon Layer
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:3391]: Why vector and polygon layer added again?

File: MAUI/Maps/migration.md [hunk 5]
```diff
@@ -2511,11 +2785,27 @@ public MainPage()
       {{'[StrokeDashArray](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html#Syncfusion_Maui_Maps_MapPolyline_StrokeDashArray)'| markdownify }}
    </td>
 </tr>
+<tr>
+   <td>
+      {{'-'| markdownify }}
+   </td>
+   <td>
+      {{'[AnimationEasing]()'| markdownify }} in {{'[MapPolylineLayer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolylineLayer.html)'| markdownify }} class
```
Comment (vigneshrameshsync) [inline MAUI/Maps/migration.md:2793]: Update snippet too with these properties

Comment (Shalini-Ashokan) [reply MAUI/Maps/migration.md:2793]: Updated sir

File: MAUI/Maps/vector-layers/circle-layer.md [hunk 6]
```diff
@@ -489,4 +489,148 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Circle stroke color](images/circle-layer/circle-stroke-color.png)
\ No newline at end of file
+![Circle stroke color](images/circle-layer/circle-stroke-color.png)
+
+## Animation
+
+You can apply animation for the [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) using the `MapCircleLayer.AnimationDuration` and `MapCircleLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/circle-layer.md:496]: You can apply animation for the [`MapCircle`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapCircle.html) using the `MapCircleLayer.AnimationDuration` and `MapCircleLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

File: MAUI/Maps/vector-layers/circle-layer.md [hunk 7]
```diff
@@ -489,4 +489,148 @@ this.Content = maps;
...
+
+By default, there will not be any animation.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/circle-layer.md:498]: By default, there will be no animation.

File: MAUI/Maps/vector-layers/line-layer.md [hunk 8]
```diff
@@ -582,4 +582,110 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Line shape dash array](images/line-layer/line_shape_dash_array.png)
\ No newline at end of file
+![Line shape dash array](images/line-layer/line_shape_dash_array.png)
+
+## Animation
+
+You can apply animation for the [`MapLine`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html) using the `MapLineLayer.AnimationDuration` and `MapLineLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/line-layer.md:589]: You can apply animation for the [`MapLine`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapLine.html) using the `MapLineLayer.AnimationDuration` and `MapLineLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

File: MAUI/Maps/vector-layers/line-layer.md [hunk 9]
```diff
@@ -582,4 +582,110 @@ this.Content = maps;
...
+
+By default, there will not be any animation.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/line-layer.md:591]: By default, there will be no animation.

File: MAUI/Maps/vector-layers/polyline-layer.md [hunk 10]
```diff
@@ -426,4 +426,90 @@ this.Content = maps;
 
 {% endtabs %}
 
-![Polyline dash array](images/polyline-layer/polyline-dash-array.png)
\ No newline at end of file
+![Polyline dash array](images/polyline-layer/polyline-dash-array.png)
+
+## Animation
+
+You can apply animation for the [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) using the `MapPolylineLayer.AnimationDuration` and `MapPolylineLayer.AnimationEasing` properties and able to customize the animation curve and duration.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/polyline-layer.md:433]: You can apply animation for the [`MapPolyline`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Maps.MapPolyline.html) using the `MapPolylineLayer.AnimationDuration` and `MapPolylineLayer.AnimationEasing` properties and be able to customize the animation curve and duration.

File: MAUI/Maps/vector-layers/polyline-layer.md [hunk 11]
```diff
@@ -426,4 +426,90 @@ this.Content = maps;
...
+
+By default, there will not be any animation.
```
Comment (florenceliliana) [inline MAUI/Maps/vector-layers/polyline-layer.md:435]: By default, there will be no animation.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

File: MAUI/Maps/getting-started.md [hunk 12]
```diff
@@ -267,6 +267,42 @@ public MainPage()
 
 ![maps basic view](images/getting-started/map_basic_view.png)
 
+## Custom shapes
+
+The MAUI Maps control allows you to render any custom shape to make a map look like building infrastructure, a sports stadium, plane or bus seat arrangements, and more using `Geometry` property.
```
[the remark this answers is not in the document]
Comment (Shalini-Ashokan) [reply MAUI/Maps/getting-started.md:272]: Updated the content

File: MAUI/Maps/vector-layers/circle-layer.md (hunk 6 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/circle-layer.md:496]: Updated the content

File: MAUI/Maps/vector-layers/circle-layer.md (hunk 7 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/circle-layer.md:498]: Updated the content

File: MAUI/Maps/vector-layers/line-layer.md (hunk 8 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/line-layer.md:589]: Updated the content

File: MAUI/Maps/vector-layers/line-layer.md (hunk 9 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/line-layer.md]: Updated the content

File: MAUI/Maps/vector-layers/polyline-layer.md (hunk 10 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/polyline-layer.md]: Updated the content

File: MAUI/Maps/vector-layers/polyline-layer.md (hunk 11 above)
Comment (Shalini-Ashokan) [reply MAUI/Maps/vector-layers/polyline-layer.md]: Updated the content

**Review by vigneshrameshsync — APPROVED** [review]
Fine.

**Pull request merged**